crates/contract/src/call.rs

```diff
@@ -252,12 +256,33 @@ impl<T: Transport + Clone, P: Provider<T, N>, N: Network> RawCallBuilder<T, P, N
     /// same as [`call_raw`](Self::call_raw).
     #[inline]
     pub fn new_raw(provider: P, input: Bytes) -> Self {
-        Self::new_inner(provider, input, ())
+        Self::new_inner_call(provider, input, ())
+    }
+
+    /// Creates a new call builder with the provided provider and contract
+    /// deploy code.
+    ///
+    /// Will not decode the output of the call, meaning that
+    /// [`call`](Self::call) will behave the same as
+    /// [`call_raw`](Self::call_raw).
+    pub fn new_raw_deploy(provider: P, input: Bytes) -> Self {
```

Comment:
Can you add the sol! NOTE comment (see new_sol) ?

Reply:
🖖